### PR TITLE
remove unused env values for testing, just URL, add setup as parameter

### PR DESCRIPTION
**Pull request by jscotka**

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: failure, return code = __1__ 
* Result url: __[https://da.gd/HJX0](https://da.gd/HJX0)__
* Comment: __I have no permission to set up status in checks__

File: Makefile [hunk 1]
```diff
@@ -34,7 +34,7 @@ run: build
 	docker run -e MYHOSTNAME=localhost -e PLAIN_AUTH $(IMAGE_REPOSITORY)
 
 test: build
-	MODULE=docker URL="docker=${IMAGE_REPOSITORY}" DOCKERFILE="../$(DOCKERFILE)" VERSION=${VERSION} DISTRO=${DISTRO} make -C tests test
+	URL="docker=${IMAGE_REPOSITORY}" make -C tests test
```
Comment (phracek) [inline Makefile:37]: Why did you delete MODULE=docker? I guess, it is prerequisite, or MODULE=docker is default?

File: tests/config.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ testdependecies:
     rpms:
         - python2-expect
         - telnet
+default_module: docker
```
Comment (phracek) [inline tests/config.yaml:14]: I see, I did not realize it. I would prefer to keep both, what do you think?

**Review by phracek — CHANGES_REQUESTED** [review]
Just fee comments, otherwise looks good to me.

File: tests/config.yaml (hunk 2 above)
Comment (phracek) [reply tests/config.yaml:14]: This needs to be documented. default_module directive is important.

File: Makefile (hunk 1 above)
Comment (jscotka) [reply Makefile:37]: no, it is connected with second change in `config.yaml` `default_module: docker`

File: tests/config.yaml (hunk 2 above)
Comment (jscotka) [reply tests/config.yaml:14]: it is important, sure and it is documented well on various places:
http://meta-test-family.readthedocs.io/en/latest/user_guide/how_to_write_conf_file.html
and in 
http://meta-test-family.readthedocs.io/en/latest/user_guide/environment_variables.html
and also in commented config:
https://github.com/fedora-modularity/meta-test-family/blob/master/docs/example-config.yaml

it is enough, or you would like to document it also on some other place?

**Review by phracek — APPROVED** [review]
